build.gradle

```diff
@@ -29,6 +30,27 @@ repositories {
     mavenLocal()
 }
 
+def props = new Properties()
+
+task copySampleProperties(type: Copy) {
+    println "Configuring copying the build.properties.sample into build.properties"
+    def sampleBuildFile = 'build.properties.sample'
+    if(new File(sampleBuildFile).exists()) { 
+        from '.'
+        into '.'
+        include sampleBuildFile
+        rename { String fileName ->
+            fileName.replace(sampleBuildFile, "build.properties")
+        }
+        doLast {
+            file("build.properties").withInputStream { props.load(it) }
+        }
+    }
+    else {
+    println("ERROR:${sampleBuildFile} does not exist") 
+    }
+}
+
 buildscript {
```

Comment:
Aren't we suppose to copy contents(/replace) from build.properties.sample to build.properties?